python/python/opm/io/deck/__init__.py

```diff
@@ -1 +1,32 @@
 from opm._common import DeckKeyword
+from opm._common import DeckItem
+
+
+def defaulted_deckitem(self):
+    return self.__defaulted(0)
+
+
+def has_value_deckitem(self):
+    return self.__has_value(0)
+
+
+def get_item_deckitem(self):
+
+    if self.is_int():
+        return self.get_int(0)
+    elif self.is_string():
+        return self.get_str(0)
+    elif self.is_double():
+        return self.get_raw(0)
```

Comment:
This is one of the reasons the `DeckItem` has not yet got a Python binding - should the `double` access return raw value or SI value? You seem to go for raw value - and I guess that is fine?

Reply:
yes I believe that if I should choose, I will go for the raw double. However, it may be an alternative to add the possibility to also get the SI double value.

```python
def get_item_deckitem(self, SI = False):

    if self.is_int():
        return self.get_int(0)
    elif self.is_string():
        return self.get_str(0)
    elif self.is_double():
        if SI:
            return self.get_SI(0)
        else:
            return self.get_raw(0)

```
and following usage
```python
        self.assertEqual( dkw_compdate[0][8].get(), 0.5)
        self.assertEqual( dkw_compdate[0][8].get(SI=True), 0.1524)
```

if SI is set to True when getting string or integer, then this is simply ignored

Would you like me to update with this changes ? I don't have a strong opinion on this.

Reply:
I have tried to add a new property 

```python
setattr(DeckItem, "si_value", get_si_double_item_deckitem)
```

```python
self.assertEqual(dkw_wconprod[0][5].value, 1.5e5)
self.assertAlmostEqual(dkw_wconprod[0][5].si_value, 4247527, delta = 1.0)
```
Unfortunately the latter test fails. The new function/property si_value returns the raw value instead of the converted.  As far as I can see, this seems to be a defect  in the C++ function

```C++
Opm::UDAValue::getSI()
```

or may be that I don't understand how to use this function.  It would be great if you @joakim-hove could have a look at this.

Reply:
The problem here is the `Opm::UDAValue::getSI()` function; I think that is doomed. So - for `double` values we can choose between SI and raw, for `UDA` only raw is available.